pyanaconda/modules/payload/live/live.py

```diff
@@ -0,0 +1,126 @@
+#
+# Kickstart module for Live payload.
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from pyanaconda.dbus import DBus
+from pyanaconda.core.signal import Signal
+
+from pyanaconda.modules.common.constants.objects import LIVE_HANDLER
+from pyanaconda.modules.common.base import KickstartBaseModule
+from pyanaconda.modules.payload.live.live_interface import LiveHandlerInterface
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+
+class LiveHandlerModule(KickstartBaseModule):
```

Comment:
I would like to decide this in another PR. I'm also considering option to merge these two but I don't want to decide anything before.

It is not a problem to rename it later.

Reply:
Well, it is not just a name of this module. You will have to rename the namespace, the object identifier, the interface class, the directory and the tests. It will affect almost every file in this pull request, so I would prefer to have it decided now.

Reply:
Then I would go the other way. Let's have separate objects for these payloads and merge them eventually later. It looks like an extra work with uncertain results.

Reply:
After our discussion we picked names LiveOS and LiveImage for handlers.